aksel.nav.no/website/app/dev/_ui/avatar/Avatar.tsx

```diff
@@ -17,39 +17,48 @@ export const avatarUrl = (avatar_id: string) => {
 export type Avatar = {
   imageSrc: string;
   name: string;
+  type: string;
   description: string;
 };
 
 export const Avatar = ({
   imageSrc,
   name,
+  type,
   showName = false,
 }: {
   /** URL to image for the avatar graphic. */
   imageSrc: string;
   /** Avatar must always have a name, as it becomes the image alt attribute. */
   name: string;
+  /** Avatar type, a name grouping. Shown in the eyebrow alongside the name */
+  type: string;
   /** The name of what the avatar represents. */
   children?: ReactNode;
-  /** Whether or not to show the name next to the avatar. */
+  /** Whether or not to show the name (and eyebrow) next to the avatar. */
   showName?: boolean;
 }) => {
   return (
     <HStack gap="space-4" align="center">
       <Image
         className={styles.avatarImage}
-        width="24"
-        height="24"
+        width="32"
+        height="32"
         alt={`Avatar for ${name}`}
         src={imageSrc}
         priority
         loading="eager"
         aria-hidden={showName}
       />
       {showName && (
-        <BoxNew asChild marginBlock="space-1 0" marginInline="space-2 0">
-          <span className={styles.avatarName}>{name}</span>
-        </BoxNew>
+        <VStack align="start">
+          <BoxNew asChild marginBlock="space-1 0" marginInline="space-2 0">
+            <Detail textColor="subtle">{type}</Detail>
+          </BoxNew>
+          <BoxNew asChild marginBlock="space-1 0" marginInline="space-2 0">
+            <BodyShort className={styles.avatarName}>{`${name}`}</BodyShort>
+          </BoxNew>
+        </VStack>
```

Comment:
Both bodyshort and detail might need to have `as="span"` here for better SR experience.